### PR TITLE
Add import command description (Contributors assistant)

### PR DESCRIPTION
**Pull request by sidolov**
## Purpose of this pull request

New import command description for Contributor assistant. 
The import command is aimed to help Magento team members simplify the development and delivery process.

## Affected DevDocs pages

whatsnew
Added the Import section to the [Contribution Guide](https://devdocs.magento.com/contributor-guide/contributing.html#import).

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by jeff-matthews** [comment]
Hi @sidolov, what happens to the original PR after the bot copies it to an internal repo?

**Comment by sidolov** [comment]
@jeff-matthews it remains open, it's just a copy of the source code. Once the internal PR will be merged the contributor's PR will be merged automatically.

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
It all looks fine but please can you add a meaningful PR description?

**Comment by sidolov** [comment]
@jeff-matthews **Markdown linting test** is failing, but I cannot see the whitespaces there

**Comment by jeff-matthews** [comment]
@sidolov, I can fix it. You need additional spaces between the bullet list marker (`-`) and the text.

**Review by BarnyShergold — APPROVED** [review]

File: src/contributor-guide/contributing.md [hunk 1]
```diff
@@ -273,6 +273,25 @@ The command merges the listed related pull requests (`xxx`, `yyy`, `zzz`) into t
...
+### Import source code to specific repository {#import}
+
+The import command provides the ability to copy a contributor's code or pull request into an internal fork.  The internal team can then proceed with additional fixes or delivery.
+
+**Command:** To import code or a pull request, a member of the Magento team controlling the pull request enters the following command:
+
+```text
+@magento import {code|pr} to {organizationName}/{repositoryName}
+```
+
```
Comment (dobooth) [inline src/contributor-guide/contributing.md:285]: I find it very helpful to have samples of these types of commands. Can you please provide a sample of each?
And I am not clear on when you would pull in a contributors code without a PR? Is this the same as "Update Branch"?

**Review by dobooth — CHANGES_REQUESTED** [review]

File: src/contributor-guide/contributing.md (hunk 1 above)
Comment (sidolov) [reply src/contributor-guide/contributing.md:299]: @dobooth examples are added

Comment (dobooth) [reply src/contributor-guide/contributing.md:285]: Good improvement. Thanks!

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Comment by dshevtsov** [comment]
@dobooth @jeff-matthews this update seems to be for internal use only. Does it really belong to the devdocs website?

**Comment by jeff-matthews** [comment]
@dshevtsov, I think it's good for us to document it publicly because external contributors will see the related bot comments and events in PRs. This doc update provides transparency for the community as well as instructions for internal contributors.

**Pull request merged**